torch/nn/modules/sparse.py

```diff
@@ -119,6 +124,14 @@ def __repr__(self):
         s += ')'
         return s.format(name=self.__class__.__name__, **self.__dict__)
 
+    @classmethod
+    def load_pretrained(cls, embeddings, freeze=True):
```

Comment:
nit: I'd call it `from_pretrained`, since it's not doing any e.g. loading from dist nor it loads the state into an existing object. It's an alternative constructor